jerry-core/debugger/debugger.h

```diff
@@ -185,6 +188,8 @@ typedef enum
   JERRY_DEBUGGER_EVAL_PART = 17, /**< next message of evaluating a string */
 
   JERRY_DEBUGGER_MESSAGES_IN_MAX_COUNT, /**< number of different type of input messages */
+  JERRY_DEBUGGER_THROW = 18,
+  JERRY_DEBUGGER_THROW_PART = 19,
```

Comment:
Where is this symbol is used?

Reply:
I've updated the python client, so these messages are used, when the string is sent.